apps/linode-marketplace-influxdb/roles/influx/tasks/main.yml

```diff
@@ -0,0 +1,59 @@
+---
+# influxdb
+
+- name: config file setup
+  template:
+    src: templates/influxdb.service
+    dest: "/etc/systemd/system/influxdb.service"
+
+- name: create docker compose directory
+  file:
+    path: /etc/docker/compose/influxdb
+    state: directory
+
+- name: docker file setup
+  template:
+    src: templates/docker-compose.yaml
+    dest: "/etc/docker/compose/influxdb/docker-compose.yml"
+
+- name: reload systemd daemon
+  systemd:
+    daemon_reload: yes
+
+- name: enable influxdb service
+  systemd:
+    name: influxdb.service
+    enabled: yes
+
+- name: start influxdb service
+  systemd:
+    name: influxdb.service
```

Comment:
```suggestion
    name: influxdb
```

Reply:
corrected in [0b8e27b](https://github.com/akamai-compute-marketplace/marketplace-apps/pull/120/commits/0b8e27bb9b6641c2719566c7c337299997d95b54)